pkg/plugins/runtime/k8s/controllers/namespace_controller_test.go

```diff
@@ -49,89 +58,160 @@ var _ = Describe("NamespaceReconciler", func() {
 				},
 			},
 		)
-
-		reconciler = &controllers.NamespaceReconciler{
-			Client:          kubeClient,
-			SystemNamespace: "kuma-system",
-			CNIEnabled:      true,
-			Log:             core.Log.WithName("test"),
-		}
-	})
-
-	It("should create NetworkAttachmentDefinition", func() {
-		// given
-		req := kube_ctrl.Request{
-			NamespacedName: kube_types.NamespacedName{
-				Namespace: "non-system-ns-with-sidecar-injection",
-				Name:      "non-system-ns-with-sidecar-injection",
-			},
-		}
-
-		// when
-		result, err := reconciler.Reconcile(req)
-		// then
-		Expect(err).ToNot(HaveOccurred())
-		// and
-		Expect(result).To(BeZero())
-
-		// when
-		nads := &v1.NetworkAttachmentDefinitionList{}
-		err = kubeClient.List(context.Background(), nads)
-		// then
-		Expect(err).ToNot(HaveOccurred())
-		// and
-		Expect(nads.Items).To(HaveLen(1))
-		Expect(nads.Items[0].Namespace).To(Equal("non-system-ns-with-sidecar-injection"))
-		Expect(nads.Items[0].Name).To(Equal("kuma-cni"))
 	})
 
-	It("should ignore system namespace", func() {
-		// given
-		req := kube_ctrl.Request{
-			NamespacedName: kube_types.NamespacedName{
-				Namespace: "kube-system",
```

Comment:
hard to see in the diff, but this was changed to `kuma-system` -- please let me know if setting `kube-system` here is intentional